sqlmesh/core/engine_adapter/redshift.py

```diff
@@ -54,6 +54,18 @@ def create_view(
             view_name, query_or_df, columns_to_types, replace, no_schema_binding=True
         )
 
+    def columns(self, table_name: TableName) -> t.Dict[str, exp.DataType]:
+        """Fetches column names and types for the target table."""
```

Comment:
Can you make a comment explaining why we did this? I could see us later wondering why we needed this instead of using the base postgres implementation.